josm-presets/de-signals-eso.xml

```diff
@@ -145,6 +145,22 @@ See https://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 						display_values="substitute signal (Zs 1);cautioness signal (Zs 7);M Board (Zs 12)"
 						de.display_values="Ersatzsignal (Zs 1);Vorsichtsignal (Zs 7);M-Tafel (Zs 12)"
 						/>
+					<multiselect key="railway:signal:traversable:type"
+						text="Traversable: action at stop"
+						de.text="Mastschild: Verhalten bei Halt"
+						values="written_default_speed;dictated_default_speed;always_on_sight;on_no_contact_on_sight"
```

Comment:
I would replace the first multiselect box by one asking for the type of "Mastschild". It is easier for mappers to identify the board by their colour(s) than by its meaning (except those who have to follow their rules as their job ;-)).

What I mean:

```xml
delimiter="|"
values="written_default_speed;dictated_default_speed|on_no_contact_on_sight;written_default_speed;dictated_default_speed|always_on_sight|on_no_contact_on_sight"
display_values="white-red-white, red-triangle on white, or pure red|white-yellow-white-yellow-white|white-black-white-black-white"
de.display_values="weiß-rot-weiß, rotes Dreieck auf weiß oder nur rot|weiß-gelb-weiß-gelb-weiß|weiß-schwarz-weiß-schwarz-weiß"
```

Reply:
Should now be replaced and be rebased. Should work with the chunks. One disadvantage could be that it's not so easy to directly exclude non-existing combinations (s.a. Ks signals with white/yellow traversable should be quite rare...), but maybe better to use. Thanks!

Reply:
If there are combinations that do not exist then you can either add the specific combinations for that single signal type (if it's only one) or add another chunk.